### DISCUSSION TITLE
Should we implement validation while working on the community event api endpoints

### DISCUSSION THREAD

**Discussion by anjyakind** [EOT-Event-Organizer-s-Toolbox/eot-codebase#101] [Q&A] [answered]
I just got started with the Update Community event api and was wondering whether we will need to write any validation e.g the request body requires an id field and at least one other field for updating a specific community event. If we should write validation what library should we use?

**Comment by 0916dhkim** [comment, marked as answer]
Let's use zod for validation on both frontend & backend 👍
